opennlp-tools/src/main/java/opennlp/tools/namefind/TokenNameFinderFactory.java

```diff
@@ -142,7 +140,7 @@ public NameContextGenerator createContextGenerator() {
     AdaptiveFeatureGenerator featureGenerator = createFeatureGenerators();
 
     if (featureGenerator == null) {
-      featureGenerator = NameFinderME.createFeatureGenerator();
+      featureGenerator = createFeatureGenerators();
```

Comment:
NameFinderME.createFeatureGenerator() has been deprecated, hence switched to the local createFeatureGenerators().